crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -43,9 +43,11 @@ impl ProtocolFee {
                 // if the quote is missing, we can't determine if the order is outside the
                 // market price so we protect the user and not charge a fee
                 let Some(quote) = quote else {
+                    tracing::trace!(?order.metadata.uid, "quote is missing, not charging a fee");
```

Comment:
This strikes me as something we should at least debug log (I thought we even had a warning log somewhere when there is a missing quote). I think we actively need to debug those cases and make sure that every order that makes it into the system has a code.

Reply:
We have it here https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/database/quotes.rs#L50-L56
I didn't know should I add this one also, but did it for completness.

Reply:
If we already have this information elsewhere, let's not log it twice. While I think we could still do a better job at adding more important debugging info, we should also not overlog the same data in multiple places.

Reply:
removed